### ISSUE TITLE
Cannot reproduce Integer quantization result using original code

### ISSUE THREAD

**Issue by Hyunseok-Kim0** [tensorflow/models#10809]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [yes] I am using the latest TensorFlow Model Garden release and TensorFlow 2.
- [yes] I am reporting the issue to the correct repository. (Model Garden official or research directory)
- [yes] I checked to make sure that this issue has not been filed already.

## 1. The entire URL of the file you are using

https://github.com/tensorflow/models/tree/master/official/projects/movinet

## 2. Describe the bug

I cannot reproduce int8 quantized movinet model with proper accuracy. My Fine-tuned float32 model shows almost same accuracy as reported in original movinet, but post training quantized int8 model always shows severely damaged accuracy.
This thing happens with original model, which means quantization itself is not working properly. Please see the attached csv file from quantization debugging result of original movinet. As you see in the csv file, global average pooling 3d layers look like the main cause of accuracy drop.
The most difficult thing to understand is that the original int8 quantized tfhub file shows quite stable performance. But the int8 tflite model in tfhub has broken structure. It does not use pool_frame_buffer and pool_frame_count, while float16 and float32 model do. Also, the model itself has slightly different layers. I guess someone didn't update it, or maybe tried to avoid accuracy drop from global pooling layer?

[quant_debugger_results.csv](https://github.com/tensorflow/models/files/9781737/quant_debugger_results.csv)

## 3. Steps to reproduce

followed quantization procedure of original repo using Kinetics 600 dataset. (movinet/tools/quantize_movinet.py)
I added code below in quantization debugger code to get the csv file.

tensor_data = [tensor_data[i['name'].replace('call_', '').split(":")[0]] for i in input_details]

https://github.com/tensorflow/tensorflow/blob/2b3b5084bb9fbd2074d09f5ccdd0491368817b52/tensorflow/lite/tools/optimize/debugging/python/debugger.py#L429

## 4. Expected behavior

Int8 quantized model with acceptable accuracy change (within few percents from original float32 model) should be generated.

## 5. Additional context

Include any logs that would be helpful to diagnose the problem.

## 6. System information

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Linux Ubuntu 20.04
- TensorFlow installed from (source or binary): from tensorflow dockerhub (docker pull tensorflow/tensorflow:2.9.2-gpu)
- TensorFlow version (use command below): 2.9.2
- Python version: 3.8.10
- CUDA/cuDNN version: 11.2 / 8
- GPU model and memory: RTX 3090 24GB

**Comment by emi-wada** [comment]
@Hyunseok-Kim0 
Hello. I want to produce int8 quantized movinet model too.
However, it doesn't proceed representative dataset(kinetics600) well and occurs error. 
How do you prepare kinetics600? In my case, I get csv and json file and set the path to this dataset's directory.
